src/tfr_executive/src/clock_service.cpp

```diff
@@ -0,0 +1,106 @@
+/*
+ * The clock service is a really simple node that exposes some utility services
+ * to simplify the operation of the executive system.
+ * 
+ * PARAMETERS 
+ * - ~driving_time: The amount of time we anticipate driving, to take in 
+ *                  seconds. (`float`, default 60)
+ * - ~dumping_time: The amount of time we anticipate dumping to take in 
+ *                  seconds. (`float`, default 30)
+ * - ~mission_time: The amount of time we allocate for the mission in total 
+ *                  seconds. (`float`, default: 600)
+ * SERVICES  
+ * 1. start_mission: starts the mission clock.
+ * 2. time_remaining: returns the amount of time remaining in the mission
+ * 3. digging_time: Gives the dumping time which is equal to  
```

Comment:
"digging_time: Gives the dumping time"? Is this correct?

Also, does this mean "the time that we should stop digging?" or does this mean "how long we should be digging?"

Reply:
(the answer to that second question determines if I have a concern about something else)

Reply:
thats a brain fart.